src/instruments/src/EFB/Dashboard/Widgets/ColorMetar.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { ColorCode, MetarParserType } from '@instruments/common/metarTypes';
+
+/**
+ * Returns HTML with coloring for METAR parts marked with coloring hints.
+ * ColorCode: src/instruments/src/Common/metarTypes.ts
+ * @param props
+ */
+const ColoredMetar = (props: { metar: MetarParserType }) => {
```

Comment:
Can you help me out and explain what exactly you mean with this?

Reply:
Destructuring is something in JavaScript that allows you to take items out of a larger collection so that you don't always have to refer to that item by specifying its parent first. For example: 

// Object declaration
const person = {
    name: 'Erick Torres',
    attendsSchool: true,
    likesBread: true,
}

//Destructure
const { attendsSchool } = Person;

//Equivalent
console.log(attendsSchool); // true
console.log(person.attendsSchool); //true

Both of these methods allow you to refer to the same piece of data, but the benefit is that you don't always need to access the property, in this case 'attendsSchool' from its container (in this case person).